bokeh/core/property_mixins.py

```diff
@@ -178,9 +178,9 @@ class _BaseLineProps(HasProps):
 
 class _BaseTextProps(HasProps):
 
-    text_font = String("helvetica", help="""
+    text_font = String("Helvetica", help="""
```

Comment:
Why is this change necessary?

Reply:
Not necessary, but currently we have a mixture of lower and upper case versions in stylesheets and defaults. CSS font face names are case insensitive, so it can be either. I changed it for the sake of consistency.

Reply:
If we are going to make things consistent, my preference is lower case throughout for font names.